api/applications_api.py

```diff
@@ -205,7 +205,7 @@ class ApplicationEventSerializer(serializers.ModelSerializer):
         help_text="List of reservation units applied for this event with priority included.",
     )
 
-    status = serializers.CharField(help_text="Status of this application")
+    status = serializers.CharField(help_text="Status of this application event")
```

Comment:
Is it possible to also document the values with this framework?

In openapi these are called enums (https://swagger.io/docs/specification/data-models/enums/).

Reply:
I'd imagine if you use ChoiceField (or MultipleChoiceField, but i guess this is just single choice)  instead of CharField those get documented? Which imo would be correct choice anyway.

Reply:
very good, I'll change that

Reply:
Looking good now 👍